packages/p2p/src/components/modal-manager/modals/business-hour-modal/business-hour-modal-edit/business-hour-modal-edit.scss

```diff
@@ -0,0 +1,139 @@
+.business-hour-modal-edit {
+    display: flex;
+    flex-direction: column;
+    gap: 1.6rem;
+
+    &__days {
+        display: flex;
+        gap: 0.8rem;
+
+        &-circle {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            min-height: 3.2rem;
+            min-width: 3.2rem;
+            border-radius: 50%;
+            background-color: #5c616d;
+            border: none;
+            cursor: pointer;
+
+            &--unselected {
+                background-color: transparent;
+                border: 1px solid rgba(0, 0, 0, 0.08);
+            }
+        }
+    }
+
+    &__selector {
+        display: flex;
+        flex-direction: column;
+        gap: 0.8rem;
+
+        &-item {
+            display: flex;
+            align-items: center;
+            gap: 0.8rem;
+
+            &-text {
+                width: 9rem;
+            }
+
+            &__dropdown {
+                background-color: var(--general-section-1);
+                padding: 0.8rem 1.6rem;
+                border-radius: 4px;
+                width: 22rem;
+                display: flex;
+                justify-content: center;
+
+                .dc-dropdown {
+                    &-container {
+                        min-width: fit-content;
+                    }
+
+                    &__container {
+                        display: flex;
+                        gap: 0.8rem;
+                        align-items: center;
+                    }
+
+                    &__display {
+                        height: unset;
+                        background: transparent;
+                        border: none;
+
+                        &-text {
+                            font-size: 1.2rem;
```

Comment:
Why do you need to define font-size here? can you use `Text` component for it?

Reply:
this is for the dropdown component itself, the display text component inside of it has a default size of 'xs' which is 14px but we need the display text to be 12px. We could resolve this if i pass props but thats more props to be added to the dropdown component and then to the display text component 😓